lib/components/narrative/narrative-itineraries.js

```diff
@@ -226,17 +270,21 @@ const mapStateToProps = (state, ownProps) => {
   const itineraries = getActiveItineraries(state.otp)
   const realtimeEffects = getRealtimeEffects(state.otp)
   const useRealtime = state.otp.useRealtime
+  const urlParams = coreUtils.query.getUrlParams()
+
   return {
-    activeSearch,
-    errors: getResponsesWithErrors(state.otp),
     // swap out realtime itineraries with non-realtime depending on boolean
-    itineraries,
-    pending,
-    realtimeEffects,
     activeItinerary: activeSearch && activeSearch.activeItinerary,
     activeLeg: activeSearch && activeSearch.activeLeg,
+    activeSearch,
     activeStep: activeSearch && activeSearch.activeStep,
+    errors: getResponsesWithErrors(state.otp),
+    filter: state.otp.filter,
```

Comment:
I'm not sure if this is needed any longer?

Reply:
Thanks, updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/3f138bb57c187373a0dea19cc90aea90f8fb92bd.